### PR TITLE
Gdr 2371

### PR DESCRIPTION
**Pull request by gladkia**
See: https://github.com/gdrplatform/gDRstyle/pull/71
Preview: https://gdrplatform.github.io/gDRimport/

### CODE REVIEW

**Review by bczech — APPROVED** [review]

File: .github/workflows/pkgdown.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+name: pkgdown
+
+on:
+  push:
+    branches: [main, master, test_ci_pkgdown]
+
+jobs:
+  trigger_pkgdown:
+    uses: gdrplatform/gDRstyle/.github/workflows/pkgdown.yaml@GDR-2371_2
```
Comment (marcin-kam) [inline .github/workflows/pkgdown.yaml:17]: Don't forget to change branch and remove DEFAULT_BRANCH

Comment (gladkia) [reply .github/workflows/pkgdown.yaml]: done.

**Review by j-smola — COMMENTED** [review]
Please, update packages version:
gDRstyle 1.1.3
gDRtestData 1.1.7
gDRutils 1.1.5

and
RoxygenNote: 7.3.0

**Comment by gladkia** [comment]
> Please, update packages version: gDRstyle 1.1.3 gDRtestData 1.1.7 gDRutils 1.1.5
> 
> and RoxygenNote: 7.3.0

Done.

**Pull request merged**